css_styles/styles.css

```diff
@@ -587,3 +614,306 @@ footer {
 .footerbtns {
   display: flex;
 }
+
+/* Desktop version */
+
+@media screen and (min-width: 768px) {
+  .mymenu {
+    visibility: hidden;
+  }
+
+  html {
+    overflow: none;
+  }
+
+  .nav-list {
+    display: flex;
+    margin-right: 206px;
+    margin-top: 26px;
+  }
+
+  .nav-list li {
+    list-style: none;
+    padding-right: 32px;
+  }
+
+  .nav-list li a {
+    color: #fff;
+  }
+
+  .nav-list li a:hover {
+    border-top: 2px solid rgb(255, 0, 0);
+    background-color: #2b2a2a;
+  }
+
+  .mylogo {
+    margin-left: 154px;
+  }
+
+  .mybiopage {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: start;
+    height: 870px;
+    width: 100%;
+    left: 0;
+    top: 0;
+    background-image: url(/images/Group-desktop.png);
+    background-repeat: no-repeat;
+    background-position: right;
+  }
+
+  .socialbtns-main {
+    height: 100%;
+    justify-items: center;
+    padding-top: 30px;
+  }
+
+  .socialbtns-main ul {
+    display: flex;
+    flex-direction: column;
+    height: 300px;
+  }
+
+  .mybio {
+    padding-left: 75px;
+    width: 1440px;
+    left: 0;
+    top: 0;
+  }
+
+  .mybio h1 {
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 78px;
+    line-height: 82px;
+    width: 470px;
+    height: 160px;
+    top: 192px;
+  }
+
+  .mybio h2 {
+    width: 566px;
+    height: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+  }
+
+  .mybio p {
+    width: 566px;
+    height: 123px;
+    font-family: "Inter", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .myworks {
+    grid-template-columns: 1fr 1fr;
+    padding: 20px 24px;
+  }
+
+  #work-headline {
+    border: none;
+    line-height: 70px;
+  }
+
+  .line {
+    width: 50%;
+    border: 1px solid black;
+  }
+
+  .multi-storybg {
+    height: 470px;
+    padding: 0;
+    margin: 0;
+    width: 50%;
+  }
+
+  .div-myworks {
+    height: auto;
+  }
+
+  .projects-section {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  #about-me {
+    background-image: url(/images/aboutme-desktop.png);
+    background-repeat: no-repeat;
+    background-position-y: center;
+  }
```

Comment:
I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

>### Your design
>
><img width="1440" alt="Screenshot 2023-05-05 at 2 52 22 PM" src="https://user-images.githubusercontent.com/91237525/236546957-92c9297b-b25f-44d9-ac40-88ea38c7af2c.png">
>
>### The Figma Design
>
><img width="732" alt="Screenshot 2023-05-05 at 3 02 37 PM" src="https://user-images.githubusercontent.com/91237525/236547129-a32b7360-0a8a-4e96-a785-26e5663f2724.png">

Reply:
I have updated my design in light of your positive suggestions please.